src/main/java/io/axoniq/axonserver/connector/event/DcbEventChannel.java

```diff
@@ -181,6 +184,63 @@ default ResultStream<StreamEventsResponse> stream(StreamEventsRequest request, i
      */
     CompletableFuture<GetSequenceAtResponse> getSequenceAt(Instant timestamp);
 
+
+    /**
+     * Schedule the given {@code event} to be published after given {@code triggerDuration}. The returned value can be
+     * used to cancel the schedule, or to reschedule the event to another time.
+     *
+     * @param triggerDuration the amount of time to wait to publish the event
+     * @param event           the event to publish
+     * @return a token used to cancel the schedule
+     */
+    default CompletableFuture<String> scheduleEvent(Duration triggerDuration, Event event) {
+        return scheduleEvent(Instant.now().plus(triggerDuration), event);
+    }
+
+    /**
+     * Schedule the given {@code event} to be published at given {@code scheduleTime}. The returned value can be used to
+     * cancel the schedule, or to reschedule the event to another time.
+     *
+     * @param scheduleTime The scheduleTime at which to publish the event
+     * @param event        The event to publish
+     * @return a token used to cancel the schedule
+     */
+    CompletableFuture<String> scheduleEvent(Instant scheduleTime, Event event);
+
+    /**
+     * Cancels the scheduled publication of an event for which the given {@code scheduleToken} was returned.
+     *
+     * @param scheduleToken the token provided when scheduling the event to be cancelled
+     * @return a future reference to the result of the instruction
+     */
+    CompletableFuture<InstructionAck> cancelSchedule(String scheduleToken);
+
+    /**
+     * Reschedules the scheduled event with the given {@code scheduleToken} and reschedule it
+     * to be published after given {@code triggerDuration}. If no event is provided, the original event will be
```

Comment:
Nit: We can clarify the fact we are not requiring an `Event` by using `@NonNull` and `@Nullable` annotations on the parameters.